sentry-core/src/scope/real.rs

```diff
@@ -204,7 +204,54 @@ impl Scope {
         Arc::make_mut(&mut self.contexts).remove(key);
     }
 
+    /// Returns a mutable reference to a context key if it exists.
+    ///
+    /// This allows in-place modification of a context value, especially useful when dealing
+    /// with [`Context::Other`] and need to do something more complicated then the
+    /// [`Scope::set_context_other`] shortcut allows.
+    ///
+    /// If the `key` does not exist `None` is returned.
+    pub fn context_mut(&mut self, key: &str) -> Option<&mut BTreeMap<String, Value>> {
+        match Arc::make_mut(&mut self.contexts).get_mut(key) {
+            Some(Context::Other(map)) => Some(map),
+            _ => None,
+        }
+    }
+
+    /// Sets a key-value pair in a custom context section.
+    ///
+    /// The context has a number of well-defined sections but also allows creating custom
+    /// sections with arbitrary key-value mappings using the [`Context::Other`] map.  This
+    /// is useful to create application-specific sections.
+    ///
+    /// This method is a shortcut which allows you to directly set such a key-value pair as
+    /// this is much more convenient than using the [`Scope::set_context`] method.  If the
+    /// section does not yet exist it is created, if the key already exists in this section
+    /// it is overwritten.
+    pub fn set_context_other(
```

Comment:
This method does not adhere to the unified specification and looks like it is just another form of `set_extra`, except that it writes into a non-standard `"other"` context, that isn't useful. The entire point of the context API is to encourage semantic names for contexts.

Let's bring this up in the next SDK evolution comittee, I'll add you to this meeting @flub.

Reply:
now i'm being told at the same time to prefer `Context::Other` over `set_extra` as well as the other way round 😕   Looking forward to the meeting 👍